modules/silicon/scripts/build/container/jupyterlab/cloudbuild.yaml

```diff
@@ -0,0 +1,38 @@
+timeout: 3600s
+steps:
+- name: 'gcr.io/cloud-builders/git'
+  args: ['clone', '-b', '2022.02.01_02.19.58', 'https://github.com/The-OpenROAD-Project/OpenLane']
+- name: 'gcr.io/cloud-builders/docker'
+  entrypoint: '/bin/bash'
+  env:
+  - EXTERNAL_PDK_INSTALLATION=0
+  - NO_PDKS=0
+  args:
+  - '-c'
+  - |-
+    apt-get update && apt install -yq python3-venv
+    python3 -m venv env
+    env/bin/python -m pip install pyyaml click
+- name: 'gcr.io/cloud-builders/docker'
+  entrypoint: '/bin/bash'
+  env:
+  - EXTERNAL_PDK_INSTALLATION=0
+  - NO_PDKS=0
+  args:
+  - '-c'
+  - |-
+    source env/bin/activate
+    make -C OpenLane openlane
+- name: 'gcr.io/cloud-builders/docker'
+  args: ['tag', 'efabless/openlane:current', 'gcr.io/$PROJECT_ID/openlane-pdk:$BUILD_ID']
```

Comment:
could use `OPENLANE_IMAGE_NAME` instead of the default image name: https://github.com/The-OpenROAD-Project/OpenLane/blob/f01acb5c2e875acac9e9a1347b251099378b7eba/docker/Makefile#L3